package.json

```diff
@@ -2,26 +2,28 @@
   "name": "@most/hold",
   "version": "2.0.0",
   "description": "Emit latest event to new observers",
-  "main": "dist/hold.js",
-  "jsnext:main": "dist/hold.es2015.js",
-  "module": "dist/hold.es2015.js",
+  "main": "dist/index.js",
+  "module": "dist/index.es.js",
+  "jsnext:main": "dist/index.es.js",
```

Comment:
In new versions you can omit `jsnext:main` which is rollup-specific and was already deprecated.